packages/api/cms-api/src/action-logs/action-logs.resolver.ts

```diff
@@ -0,0 +1,61 @@
+import { EntityManager, PostgreSqlDriver } from "@mikro-orm/postgresql";
+import { ForbiddenException, Inject } from "@nestjs/common";
+import { Args, ID, Query, Resolver } from "@nestjs/graphql";
+
+import { GetCurrentUser } from "../auth/decorators/get-current-user.decorator";
+import { gqlArgsToMikroOrmQuery } from "../common/filter/mikro-orm";
+import { RequiredPermission } from "../user-permissions/decorators/required-permission.decorator";
+import { CurrentUser } from "../user-permissions/dto/current-user";
+import { ACCESS_CONTROL_SERVICE } from "../user-permissions/user-permissions.constants";
+import { AccessControlServiceInterface } from "../user-permissions/user-permissions.types";
+import { ACTION_LOGS_METADATA_KEY, ActionLogMetadata } from "./action-logs.decorator";
+import { PaginatedActionLogs } from "./dto/paginated-action-logs";
+import { PaginatedActionLogsArgs } from "./dto/paginated-action-logs.args";
+import { ActionLog } from "./entities/action-log.entity";
+
+@RequiredPermission("actionLogs", { skipScopeCheck: true })
+@Resolver(() => ActionLog)
+export class ActionLogsResolver {
```

Comment:
ActionLogsResolver are missing in the `@comet/cms-api` - development schema. 

We need to add the ActionLogsResolver to the `packages/api/cms-api/generate-schema.ts` so they will get  generated: https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/generate-schema.ts#L128

Reply:
fixed